CHANGELOG.md

```diff
@@ -2,6 +2,13 @@ vim Cookbook CHANGELOG
 ======================
 This file is used to list changes made in each version of the vim cookbook.
 
+v1.1.4 (2015-09-04)
```

Comment:
Instead of 1.1.4 make this UNRELEASED and I'll bump it up to the version when we do the release